### PR TITLE
[ISSUE #3390] "getExtension()" can return null.[BatchPublishMessageProcessor]

### LINKED ISSUE #3390: [Enhancement] "getExtension()" can return null.[BatchPublishMessageProcessor]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java line 106

### Describe the solution you'd like

Use 'Objects. requireNonNull()' to avoid NPE.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by nikam14**
Fixes #3390 .

### Modifications

imported the Objects class and
added Objects.requireNonNull().

### Documentation

- Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by nikam14** [comment]
@Alonexc can you help the test cases are failing

**Comment by mroccyen** [comment]
image

@nikam14 please fix checkstyle

**Review by mxsm — COMMENTED** [review]
@nikam14  Import eventmesh-code-style.xml into develop tool. and format code to fix ci not pass. you can refer to:https://eventmesh.apache.org/community/contribute/contribute

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java [hunk 1]
```diff
@@ -103,7 +104,7 @@ public void process(BatchMessage message, EventEmitter<Response> emitter) throws
 
         for (CloudEvent event : cloudEvents) {
             String seqNum = event.getId();
-            String uniqueId = (event.getExtension(ProtocolKey.UNIQUE_ID) == null) ? "" : event.getExtension(ProtocolKey.UNIQUE_ID).toString();
+            String uniqueId = (event.getExtension(ProtocolKey.UNIQUE_ID) == null) ? "" : Objects.requireNonNull(event).getExtension(ProtocolKey.UNIQUE_ID).toString();
```
Comment (Alonexc) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java:107]: This line is too long and causes the checkstyle to not pass, so you can change the line.

**Comment by nikam14** [comment]
@Alonexc @mxsm @mroccyen thanks your guidance.

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java (hunk 1 above)
Comment (nikam14) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/processor/BatchPublishMessageProcessor.java:107]: changed the line in this [pr](https://github.com/nikam14/incubator-eventmesh/pull/1). 
waiting for your respose to make PR to main repo.

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**